src/partition/ReceivedBoundingBox.cpp

```diff
@@ -118,10 +119,17 @@ void ReceivedBoundingBox::computeBoundingBox()
       if (compareBoundingBox(_bb,other_rank.second)) {
         _feedback.push_back(other_rank.first);            
       }
-    }   
+    }
+
+    // to prevent sending empty vector!
+    if(_feedback.size() == 0)
```

Comment:
I think it's better to simply have an if-block around the send below and sending the size before. It's minimally less efficient, but using dummy data is dangerous.

Reply:
I agree

Reply:
ok, I changed it!